Domain/converterInterface.go

```diff
@@ -0,0 +1,6 @@
+package Domain
+
+type ConverterInterface interface {
+	FromViewModel(model UserViewModelInterface) UserApplicationModelInterface
```

Comment:
look something like https://github.com/devfeel/mapper  to solve that problem.

Reply:
Oh, it is interesting solution, thanks